plugins/extra/p2js/componentEditors/P2BodyEditor.ts

```diff
@@ -61,6 +62,12 @@ export default class P2BodyEditor {
       this.editConfig("setProperty", "height", parseFloat(event.target.value));
     });
 
+    this.angleRow = SupClient.table.appendRow(this.tbody, SupClient.i18n.t("componentEditors:P2Body.angle"));
```

Comment:
No it should fallback to english automatically. Do you see it being shown as "componentEditors:P2Body.angle" somewhere?